### PR TITLE
Rephrasing 2i1c+

### PR DESCRIPTION
**Pull request by hanwu85**
To reduce unnecessary wording that might be confusing.

### CODE REVIEW

**Review by Nanush7 — APPROVED** [review]

**Review by lgarron — COMMENTED** [review]
Thanks for starting the PR! Some thoughts in a comment.

File: wca-guidelines.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ To be more informative, each Guideline is classified using one of the following
 - 2i++) [ADDITION] The organization team may provide the competitor an unofficial stopwatch for viewing the elapsed time (started together with the main stopwatch), in which case the competitor is not permitted to touch the official stopwatch.
 - 2i+++) [REMINDER] Bluetooth puzzles are considered electronic devices.
 - 2i1b+) [CLARIFICATION] This includes relevant devices which are switched off or disconnected.
-- 2i1c+) [CLARIFICATION] Electronic hand warmers may be used before or after an attempt only. Non-electronic hand warmers may be used at any time during an attempt.
+- 2i1c+) [CLARIFICATION] Electronic hand warmers are still considered electronic devices. Non-electronic hand warmers may be used at any time during an attempt.
```
Comment (lgarron) [inline wca-guidelines.md]: Given that people may refer to this without the corresponding Regulation (2i) as context, I'd suggest something more like this.

```suggestion
- 2i1c+) [CLARIFICATION] Electronic hand warmers are considered electronic devices, and therefore are not permitted while inspecting or solving. However, non-electronic hand warmers may be used at any time during an attempt.
```

This duplicates the "inspecting or solving" part of 2i, but I think that's worth it for clarity.

Comment (Nanush7) [reply wca-guidelines.md]: LGTM.

Comment (hanwu85) [reply wca-guidelines.md]: It's a nice wording in my opinion!

Comment (Nevseros) [reply wca-guidelines.md]: LGTM also!

**Pull request merged**